File: pro-language-droid/cpp-prog/struct.cpp
#include<stdio.h>
#include<bits/stdc++.h>
using namespace std;

struct Person 
{
    int id;
    int age;
    char name[50];
};

int main()
{
    struct Person person;
    person.id = 20;
    person.age = 35;
    strcpy(person.name,"John");

    cout<<"Id : "<<person.id<<endl;
    cout<<"Age : "<<person.age<<endl;
    cout<<"Name : "<<person.name<<endl;

}

File: pro-language-droid/csharp-prog/struct.cs
using System;

struct Person{
    public string firstName;
    public string lastName;
    public int age;
    public int id;
};


class Program{
    
        static void Main(string[] args){
        
        Person person1;
        person1.firstName="john";
        person1.lastName="hall";
        person1.age=25;
        person1.id=12;

        Console.WriteLine("first Name : {0}",person1.firstName);
        Console.WriteLine("last Name : {0}",person1.lastName);
        Console.WriteLine("Age : {0}",person1.age);
        Console.WriteLine("id : {0}",person1.id);
               
        }
}
